### PR TITLE
Improve txt-suite command-line tool output

### PR DESCRIPTION
**Pull request by zaolin**
* More compact command-line output
* Adding test numbers
* Remove document information
* Remove useless newlines 

Signed-off-by: Philipp Deppenwiese <zaolin@das-labor.org>

### CODE REVIEW

File: cmd/txt-suite/main.go [hunk 1]
```diff
@@ -66,28 +66,29 @@ func run(tests []*test.Test) bool {
...
-			fmt.Printf("%-20s\n", a.Bold(a.Yellow(tests[index].Result)))
+			fmt.Printf("%-20s", a.Bold(a.Yellow(tests[index].Result)))
 		} else {
-			fmt.Printf("%-20s\n", a.Bold(a.Red(tests[index].Result)))
+			fmt.Printf("%-20s", a.Bold(a.Red(tests[index].Result)))
 		}
 		if tests[index].ErrorText != "" {
-			fmt.Printf(" %s\n", tests[index].ErrorText)
-		}
-		if tests[index].ErrorTextSpec != "" {
```
Comment (PatrickRudolph) [inline cmd/txt-suite/main.go:89]: ErrorTextSpec has been removed, but no reason was given why.

Comment (zaolin) [reply cmd/txt-suite/main.go:89]: Output looks too complex. Document ids can be looked up via testplan

Comment (PatrickRudolph) [reply cmd/txt-suite/main.go:89]: too complex? Also the testplan isn't part of the binary.
How about adding a new flag (-v for verbose) and only print if if -v is specified.

Comment (zaolin) [reply cmd/txt-suite/main.go:89]: We already have a -m option which prints all necessary information.

Comment (PatrickRudolph) [reply cmd/txt-suite/main.go:89]: ????????????

**Pull request merged**